smart-contracts/test/test_txSignatureAggregation.js

```diff
@@ -186,13 +176,13 @@ describe("Gas Cost Tests", function () {
  * 
  * 
 Unlock Gas Cost With 4 Validators
-tx0  182434
+tx0  165769
 ~~~~~~~~~~~~
-Total:  182434
+Total:  165769
```

Comment:
Are these numbers up to date? I'm wondering if they changed around after the double pegging logic was added.

Reply:
Yes, they were up to date before I added `tokenName`, `tokenSymbol`, and `tokenDecimals` to `claimData`.
The new values are 173990 gas for lock and 179737 gas for mint (I updated the comments again).